schema/schema.py

```diff
@@ -120,6 +123,8 @@ def clean_text(text: str) -> str:
     """Clean up a string of text to be formatted for the JSON schema."""
     if not text:
         return ""
+    ansi_regex = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
```

Comment:
Could this be a constant at the file root instead? I'm sure that Python will optimize this out so its not calling compile on the regular expression every time this method is called, but just wanted to be safe